script/release.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+const { versionBump, currentVersion }  = require('./version-utils');
+const { logger, spawnOrFail, prompt, shouldContinuePrompt, quit, process, path } = require('./cli-utils');
+
+const deployDemo = (version) => {
+  const demo_name = `chime-sdk-demo-${version.replace(/\./g, "-")}`;
+  logger.log(`Deploying ${demo_name} ...`);
```

Comment:
can we log the version here too

Reply:
the version is in the demo name already right?